src/components/accounts/SetRecoveryInfoContainer.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Container, Loader, Grid, Dimmer, Header } from 'semantic-ui-react'
+
+import NearInfo from './NearInfo'
+
+import styled from 'styled-components'
+
+const CustomContainer = styled(Container)`
+   && .disclaimer {
+      margin-left: 1rem;
+      margin-right: 1rem;
+      margin-top: 100px;
+
+      > .column {
+         padding-left: 0px;
+         padding-right: 0px;
+      }
+   }
+   .disclaimer-info {
+      font-weight: 600;
+      letter-spacing: 2px;
+   }
+
+   &&& .creating-info {
+      padding-right: 0px;
+      padding-top: 48px;
+
+      h1 {
+         color: #4a4f54;
+         padding-bottom: 24px;
+      }
+   }
+
+   @media screen and (max-width: 767px) {
+      &&& .creating-info {
+         padding-right: 1rem;
+      }
+   }
+`
+
+/* eslint-disable jsx-a11y/accessible-emoji */
+const SetRecoveryInfoContainer = ({ loader, children, location }) => (
+   <CustomContainer>
+      <Grid className=''>
+         <Dimmer inverted active={loader}>
+            <Loader />
+         </Dimmer>
+
+         <Grid.Row className='creating-info'>
+            <Grid.Column computer={8} tablet={8} mobile={16} className=''>
+               <Header as='h1'>Protect your Account 🔒</Header>
+               <Header as='h2'>
+                  Connect account to phone number – save recovery key on our servers.
+               </Header>
+               <Header as='h3' className='color-blue'>
+                  You can skip this if you plan to backup account keys manually.
+                  We won't be able to help you with account recovery otherwise.
```

Comment:
Is this text from the official UX? I think the sentence should read as "You can skip this blah blah... If you do skip this, we won't be able to help with your account recovery"

Reply:
There is no official UX for now, going to update text after comments from Jake and Erik